sdk/resourcemanager/azure-resourcemanager-eventhubs/CHANGELOG.md

```diff
@@ -1,5 +1,13 @@
 # Release History
 
+## 2.27.0-beta.1 (2023-04-23)
+
+### Other Changes
+
+#### Dependency Updates
+
+- Updated `api-version` to `2.27.0-beta.1`.
```

Comment:
still not correct.

OK, make sure you understand what is api-verion.

https://github.com/Azure/azure-rest-api-specs/tree/main/specification/eventhub/resource-manager#tag-package-2022-10-preview

On path to swagger, as well as swagger info, there is api-version
https://github.com/Azure/azure-rest-api-specs/blob/main/specification/eventhub/resource-manager/Microsoft.EventHub/preview/2022-10-01-preview/eventhubs.json#L6
that follow `year-month-day` or `year-month-day-preview`

Reply:
I see, now fixed in the new version.